source/_docs/z-wave/controllers.markdown

```diff
@@ -120,6 +120,6 @@ You should also check the README for details on the overlays. You might find it
 
 <div class='note'>
 
-  If you've installed the Z-Way software, you'll need to ensure you disable it before you install Home Assistant or you won't be able to access the board. Do this with `sudo /etc/init.d/z-way-server stop; sudo update-rc.d z-way-server disable`.
+  If you've installed the Z-Wave.Me Z-Way software, you can use the [Z-Wave.Me](/integrations/zwave_me). In order to use Z-Wave JS instead of Z-Way, you'll need to ensure you disable it before you install Home Assistant or you won't be able to access the board. Do this with `sudo /etc/init.d/z-way-server stop; sudo update-rc.d z-way-server disable`.
```

Comment:
This part of the documentation is about our Z-Wave JS integration, which we recommend on using. Therefore, it should be the first priority to mention.

```suggestion
  If you've installed the Z-Wave.Me Z-Way software. In order to use Z-Wave JS instead of Z-Way, you'll need to ensure you disable it before you install Home Assistant, or you won't be able to access the board. Do this with `sudo /etc/init.d/z-way-server stop; sudo update-rc.d z-way-server disable`. Alternatively, you could use the [Z-Wave.Me](/integrations/zwave_me) integration.
```